### PR TITLE
Rename 'Fuel for OpenStack' to 'Mirantis OpenStack'

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

File: pdf/pdf_install.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
   .. cssclass:: header-table
   
   +-------------------------------------+-----------------------------------+
-  | Fuel for OpenStack v3.2             |                .. cssclass:: right|
+  | Mirantis OpenStack v3.2             |                .. cssclass:: right|
```
Comment (mihgen) [inline pdf/pdf_install.rst:7]: this also needs to be 3.2.1

**Comment by mihgen** [comment]
+1
I've copied 3.2 docs, so it will not change now. Updated content will appear at http://docs.mirantis.com/fuel/master/ for now.

**Comment by mihgen** [comment]
There were some errors in build: http://fuel-docs.vm.mirantis.net:8080/job/build_docs/596/console

**Comment by e0ne** [comment]
It's known issues now

**Pull request merged**